.github/workflows/typescript-untouched-files-check.yml

```diff
@@ -0,0 +1,45 @@
+name: Typescript Untouched File Check
+
+on:
+  pull_request:
+  push:
+    branches:
+      - dependency-updates
```

Comment:
❓ : Should this action only run on `dependecy-updates`? It's my understanding it should run on any branch going to an upper envs.

Reply:
Good catch, will update. 
It seemed to have been working even with that hardcoded branch name. Even our Typescript count Github Action has dependency-updates hardcoded. Weird, will look into it. (Not too familiar with Github Action templates)